tests/baselines/reference/arrowFunctionWithObjectLiteralBody1.js

```diff
@@ -2,4 +2,4 @@
 var v = a => <any>{}
 
 //// [arrowFunctionWithObjectLiteralBody1.js]
-var v = function (a) { return {}; };
+var v = function (a) { return ({}); };

```

Comment:
it is, the parens is coming from the transformation of the cast, then we transform the lambda, and they stick. i would say this is a won't fix.